src/app/EventManagement.cpp

```diff
@@ -359,6 +360,10 @@ CHIP_ERROR EventManagement::ConstructEvent(EventLoadOutContext * apContext, Even
 
     // Callback to write the EventData
     ReturnErrorOnFailure(apDelegate->WriteEvent(apContext->mWriter));
+    if (!(apOptions->mFabricIndex.IsNull()))
+    {
+        apContext->mWriter.Put(TLV::ProfileTag(kEventManagementProfile, kFabricIndexTag), apOptions->mFabricIndex.Value());
```

Comment:
Worth documenting that this TLV is not going on the wire directly, so the fact that we're making up some random profile tag is ok because we are the only ones who will see it.

Reply:
Update document, thanks